plugins/modules/openssl_pkcs12.py

```diff
@@ -51,6 +51,13 @@
     type: list
     elements: path
     aliases: [ca_certificates]
+  other_certificates_content:
+    description:
+      - List of other certificates to include.
+      - Mutually exclusive with O(other_certificates).
```

Comment:
The documentation of `other_certificates` should say that it's mutually exclusive with this option.